libs/community/langchain_community/llms/google_palm.py

```diff
@@ -159,5 +348,13 @@ def get_num_tokens(self, text: str) -> int:
         Returns:
             The integer number of tokens in the text.
         """
+        if self.is_gemini:
+            raise ValueError("Counting tokens is not yet supported!")
         result = self.client.count_text_tokens(model=self.model_name, prompt=text)
         return result["token_count"]
+
+
+class GooglePalm(GoogleGenerativeAI):
+    """`GooglePalm` retriever alias for backwards compatibility.
+    DEPRECATED: Use `GoogleGenerativeAI` instead.
+    """
```

Comment:
It would be useful to add a deprecation warning on initialization.

Reply:
Done